src/pages/home/HomePage.js

```diff
@@ -122,6 +125,24 @@ class App extends React.Component {
         this.toggleHamburger();
     }
 
+    /**
+     * Method called when we click the floating action button
+     * will trigger the animation
+     */
+    toggleFab() {
+        const animationFinalValue = this.state.isFabActive ? 0 : 1;
+        this.setState(state => ({
+            isFabActive: !state.isFabActive,
+        }));
+
+        Animated.timing(this.animationFabActivate, {
+            toValue: animationFinalValue,
+            duration: 400,
+            easing: Easing.ease,
```

Comment:
What are the easing options we have here? Could we try something like `ease-in-out`?

Reply:
`Easing.inOut` is a helper function used to modify others easing functions, it takes another easing as a parameter. I won't lie I am not 100% sure how it works. I just pushed a new commit changing the `Easing.ease` to `Easing.inOut(Easing.ease)`, I did like the result better. I also fixed the other reviews.

You didn't mention it but we can use different easing curves to style different parameters if necessary. But it would require a new Animated.Value, which would be passed down in addition to the existing one. 

But I did like the new result with the new Easing. The [docs](https://reactnative.dev/docs/easing) have some examples on other curves. Maybe another one that could work would be `Easing.sin`, but this `inOut` is probably better, maybe change the easing function called in it.

**Easing.inOut(Easing.ease)**
![in-out](https://i.ibb.co/M55X09z/ease-in-out.gif)

Reply:
Ease in/out looks great to me! 

But it looks like the background color animation is animating from green to gray, and then it goes transparent. To fix this, can we actually have the animation change the color from `themeColors.buttonSuccessBG` to `themeColors.sidebar`? Hopefully this creates a smooth transition from green to "transparent"

Reply:
Good point. Do you want me to also change the `colors.white` to maybe `themeColors.componentBG` or `themeColors.appBG`? That would remove the necessity of the `colors` import.

Reply:
Ah yes, `componentBG` sounds great for that. Thanks!